github/ci/prow-deploy/files/jobs/kubevirt/kubevirt.github.io/kubevirt-github-io-postsubmits.yaml

```diff
@@ -0,0 +1,42 @@
+postsubmits:
+  kubevirt/kubevirt.github.io:
+    - name: publish-img
+      always_run: false
+      run_if_changed: "Makefile"
+      annotations:
+        testgrid-create-test-group: "false"
+      decorate: true
+      max_concurrency: 1
+      labels:
+        preset-dind-enabled: "true"
+        preset-kubevirtci-quay-credential: "true"
+      cluster: phx-prow
```

Comment:
For new jobs you should use `prow-workloads`, Then you also need to remove the `nodeSelector` as that doesn't exist there.

Reply:
I have no idea what this means